ui/scripts/movie.js

```diff
@@ -40,11 +42,14 @@ export default class Movie {
         const imageElem = `<img class="movie-image-cover" src=${this.cover_url} alt="Watch this space, cover is comming soon" />`;
         const cover = this.createMovieElemHTML('figure', 'movie-image', imageElem);
 
-        const imgCaption = this.createMovieElemText("figcaption", "movie-image-caption", "See the trailer");
-        cover.appendChild(imgCaption);
-
+        
         // @todo: will need to pass url to the trailer later
-        cover.addEventListener("click", () => new Modal(this.title).fillInModal());
+        if (this.trailer_url) {
+            const imgCaption = this.createMovieElemHTML("figcaption", "movie-image-caption", "Trailer &#10162;");
```

Comment:
you could add a comment here explaining that the html character code is an arrow directed to the right (it's not obvious).

Reply:
Done!